### ISSUE TITLE
"Half-Life 2: Update" does not launch because of missing shared library

### ISSUE THREAD

**Issue by edwargix** [flathub/com.valvesoftware.Steam#69]
"Half-Life 2: Update", a linux-compatible-game, cannot launch. Here is the relevant error message from the terminal:

~/.var/app/com.valvesoftware.Steam/data/Steam/steamapps/common/Half-Life 2 Update/hl2_linux: error while loading shared libraries: libtcmalloc_minimal.so.4: cannot open shared object file: No such file or directory

**Comment by TingPing** [comment]
This is an issue that should be reported to them.

**Comment by mcepl** [comment]
What happens if you install **32bit version** of ``gperftools`` for your distro? (``gperftools.i686`` for Fedora/RHEL)

**Comment by edwargix** [comment]
I am using Gentoo and have the abi_x86_32 USE flag on for google-perftools (this enables 32-bit support for the package) and the problem persists. I don't understand flatpak very well, but isn't this fact sort of irrelevant since flatpak manages all of its libraries independently?

**Comment by TingPing** [comment]
>  I don't understand flatpak very well, but isn't this fact sort of irrelevant since flatpak manages all of its libraries independently?

Yes.

**Comment by nanonyme** [comment]
Was this ever reported upstream?

**Comment by edwargix** [comment]
Yes; I began a forum thread for it [here](https://forums.blackmesasource.com/index.php/Thread/44971-Ship-Black-Mesa-with-libtcmalloc-minimal-on-Linux/). I haven't tried to launch the game in a while and thus don't know whether it's been fixed.

**Comment by ZakMcKrack3n** [comment]
Its already included, but hl2.sh script is not preloading it.
Try launching with:
```
LD_LIBRARY_PATH="/home/myUserName/.local/share/Steam/steamapps/common/Half-Life 2 Update/bin/":${LD_LIBRARY_PATH} %command%
```
Be aware that you cant use ~ in the launch window with LD_LIBRARY_PATH (did not work for me), adjust steam install folder path accordingly.

Tested with arch linux running steam-runtime.
